### PR TITLE
Fixes alt click not opening labcoat's inventory

### LINKED ISSUE #3016: Alt-Click no longer opens the inventory of a labcoat
## Round ID:

N/A (Downstream)
## Testmerges:

N/A

## Reproduction:

- Alt-click on labcoat

Alt-Click can be used on items to open up their inventory. This is no longer possible with lab-coats and  click dragging onto yourself is honestly a pain

Probably related to https://github.com/BeeStation/BeeStation-Hornet/pull/2976

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Fixes #3016
Alt click on a labcoat will no longer toggle the labcoat style, but will instead open the inventory.
To change the style, the toggle suit style verb can be used instead which is located on the context menu, or in the object tab of the stat panel.
image

## Why It's Good For The Game

Makes it much easier to access items that are stored within a lab coat, due to conflicting key binds suit style adjustment have been moved to the object tab of the stat panel.

## Testing Photographs and Procedure

image

image

## Changelog
:cl:
tweak: Alt clicking on suits that contain embedded storage (pockets) will now open the inventory instead of adjusting the style of that article of clothing. Adjusting the style can be done through the context menu or the stat panel under the object tab.
/:cl:

### CODE REVIEW

**Review by ike709 — DISMISSED** [review]

**Comment by itsmeow** [comment]
Please merge

File: code/modules/clothing/suits/toggles.dm [hunk 1]
```diff
@@ -94,6 +94,8 @@
 //Toggle exosuits for different aesthetic styles (hoodies, suit jacket buttons, etc)
 
 /obj/item/clothing/suit/toggle/AltClick(mob/user)
+	if(SEND_SIGNAL(src, COMSIG_CLICK_ALT, user))
+		return
 	if(!user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
 		return
 	else
```
Comment (PowerfulBacon) [inline code/modules/clothing/suits/toggles.dm]:
```suggestion
	SEND_SIGNAL(src, COMSIG_CLICK_ALT, user)
```

File: code/modules/clothing/suits/toggles.dm [hunk 2]
```diff
@@ -94,15 +94,15 @@
 //Toggle exosuits for different aesthetic styles (hoodies, suit jacket buttons, etc)
 
 /obj/item/clothing/suit/toggle/AltClick(mob/user)
-	if(!user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
-		return
-	else
+	SEND_SIGNAL(src, COMSIG_CLICK_ALT, user)
 		suit_toggle(user)
```
Comment (PowerfulBacon) [inline code/modules/clothing/suits/toggles.dm]:
```suggestion
```

**Review by itsmeow — APPROVED** [review]
Looks good given that this doesn't break any other Alt+Clicking in your testing

**Review by ike709 — APPROVED** [review]

**Pull request merged**